community/contributing/docs/reviewing-prs.md

```diff
@@ -0,0 +1,69 @@
+
+# Before you begin
+
+Before reviewing documentation, do the following:
+
+Read the [documentation style guide](https://www.spinnaker.io/community/contributing/docs/docs-style-guide/) so that you can leave informed feedback.
+
+1. Review the [code of conduct](https://github.com/spinnaker/spinnaker.github.io/blob/master/community/contributing/code-of-conduct.md) and ensure that you abide by it at all times.
+2. Be polite, helpful, and considerate of others.
+3. Comment on positive aspects of a PR, as well as changes.
+4. Be empathetic and mindful of how your changes or review may be received.
+5. Ask clarifying questions if something in a PR is unclear.
+6. Assume positive intent.
+7. If you are an experienced contributor, consider pairing with new contributors whose work requires extensive changes.
+8. Be mindful of the needs of contributors that are neurodivergent or have a learning disability. Ask your fellow contributors how they work best and what communication style they prefer.
+
+# Start the review process
+
+1. Go to (<https://github.com/spinnaker/spinnaker.github.io/pulls>) to see a list of all current open PRs in the Spinnaker repository.
+
+2. Filter the open PRs using one or all of the following labels:
+
+  - Beginner friendly
+  - Help wanted
+  - Questions
+  - no-lifecycle
+
+    If a PR is labelled `draft PR` it means that the contributor would like feedback on their PR throughout the review process. Reviewers should encourage new contributors to use **draft PRs** rather than open a pull request.
+
+  - Once you've found a PR to review, understand the change by:
+
+    - Reading the PR description to learn why the PR was made, and read any linked issues attached to the PR
+    - Reading comments left by other reviewers
+    - Clicking the **Files changed** tab to see any files and lines changed
+
+3. Go to the **Files changed** tab and begin your review
+
+  - Click on the + symbol beside the line you want to comment on.
+  - Fill in any comments you have about the line in question, and click either, **Add a single comment** (if you have only one comment to make on the PR) or **Start a review** (if you have multiple comments)
+  - When you've finished reviewing the PR, click **Review changes** at the top of the page. Here, you can add a summary of your review (and leave some positive comments and thanks for the contributor!), approve the PR, comment or request changes as needed. New contributors should always choose **Comment**.
+
+# Reviewing checklist
+
+When reviewing a PR, use the following questions as a starting point:
+
+- Are there any obvious errors in grammar or language?
+- Is there a better way to phrase something?
+- Are there words, phrases, or terms that should be replaced with a non-discriminatory alternative?
+- Does the word choice, punctuation, and capitalization follow the [style guide](https://www.spinnaker.io/community/contributing/docs/docs-style-guide/)?
```

Comment:
Please make the link to the Documentation Style Guide a relative one instead of a hard-coded URL.

Reply:
Done!